### PR TITLE
Add test cases for Support-Notifications transmission and cleanup

### LINKED ISSUE #108: Create Support Notifications V2 API Test Cases - Positive
Design support-notifications positive testcases based on [support-notifications V2 API](https://github.com/edgexfoundry/edgex-go/blob/master/api/openapi/v2/support-notifications.yaml). Reference: [blackbox-testing support-notifications.postman_collection](https://github.com/edgexfoundry/blackbox-testing/blob/master/bin/postman-test/collections/support-notifications.postman_collection.json).

### PR DESCRIPTION
**Pull request by cherrycl**
Add test cases for Support-Notifications transmission and cleanup

Fix #108 #109
Signed-off-by: Cherry Wang <cherry@iotechsys.com>

### CODE REVIEW

File: TAF/testScenarios/functionalTest/V2-API/support-notifications/cleanup/DELETE.robot [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Suite Teardown  Run Teardown Keywords
+Force Tags  Skipped
+
+*** Variables ***
+${SUITE}          Support Notifications Cleanup Test Cases
+${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/support-notifications-cleanup.log
+${url}            ${supportNotificationsUrl}
+
+*** Test Cases ***
+# /transmission/age/{age}
```
Comment (jinlinGuan) [inline TAF/testScenarios/functionalTest/V2-API/support-notifications/cleanup/DELETE.robot:15]: Should be `/cleanup`

File: TAF/testScenarios/functionalTest/V2-API/support-notifications/transmission/GET-Positive.robot [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+TransmissionGET002 - Query all transmissions that transmissions are more then 20
+    # Generate more than 20 transmissions
+    Given Create Subscriptions
+    And Create Notifications Contain Category Or Label Has Subscribed
+    When Query All Transmissions
+    Then Should Return Status Code "200" And transmissions
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+    And All Transmission Should Be Found
```
Comment (jinlinGuan) [inline TAF/testScenarios/functionalTest/V2-API/support-notifications/transmission/GET-Positive.robot:35]: Only 20 transmissions should be found

**Review by jinlinGuan — CHANGES_REQUESTED** [review]

File: TAF/testScenarios/functionalTest/V2-API/support-notifications/cleanup/DELETE.robot (hunk 1 above)
Comment (cherrycl) [reply TAF/testScenarios/functionalTest/V2-API/support-notifications/cleanup/DELETE.robot]: Fixed.

File: TAF/testScenarios/functionalTest/V2-API/support-notifications/transmission/GET-Positive.robot (hunk 2 above)
Comment (cherrycl) [reply TAF/testScenarios/functionalTest/V2-API/support-notifications/transmission/GET-Positive.robot]: Fixed.

**Review by jinlinGuan — APPROVED** [review]
LGTM

**Pull request merged**